tastyworks/models/trading_account.py

```diff
@@ -90,6 +90,90 @@ async def get_remote_accounts(cls, session) -> List:
 
         return res
 
+    async def get_balance(session, account):
```

Comment:
since this is an instance method, would that require the use of `self` as a first argument?

Reply:
Just noticed that now, good point

Reply:
Question - just being nosy - should these be class methods instead since the only variables referenced are passed as arguments? Neither this method or the ones below reference self.  My other thought would be to reference self, in all of these methods?  For example adding something like this could be a quick fix:

```
async def get_balance(self, session, account):
  session = session if session else self.session
  account = account if account else self.account
  ...
```

Reply:
@boyan-soubachov please see https://github.com/boyan-soubachov/tastyworks_api/pull/49